proto/parca/query/v1alpha1/query.proto

```diff
@@ -224,6 +224,12 @@ message QueryRequest {
 
   // filter_query is the query string to filter the profile samples
   optional string filter_query = 6;
+
+  // disable_trimming disables trimming of insignificant nodes in the report
+  optional bool disable_trimming = 7;
+
+  // node_trim_threshold is the threshold for trimming insignificant nodes in the report
```

Comment:
From this comment it is unclear to me that this is a fraction or percentage. By this comment I would expect this to be a total value, which I don't think is true.